src/components/Admin/Projects/index.js

```diff
@@ -219,6 +251,25 @@ export default class Projects extends React.Component {
             },
           }).then(({data}) => {
             self.props.data.refetch()
+            project.extensions.map(function(projectExtension){
+              if(projectExtension.extension.key === "kubernetesloadbalancers" && projectExtension.environment.id === env.id) {
```

Comment:
I assume we filter on this because we want to automatically apply and create the load balancers for each project which has a load balancer installed on them? We are able to do this because we separated out the roles of route53 and LB, correct?